app/domain/authentication/authn_k8s/k8s_object_lookup.rb

```diff
@@ -59,7 +59,8 @@ def options
           ssl_options: {
             cert_store: @cert_store,
             verify_ssl: OpenSSL::SSL::VERIFY_PEER
-          }
+          },
+          http_proxy_uri: ENV['https_proxy'] ? ENV['https_proxy'] : ENV['http_proxy']
```

Comment:
Since we're treating `ENV['https_proxy']` as a truthy value anyway, this can be simplified to:

```ruby
          http_proxy_uri: ENV['https_proxy'] || ENV['http_proxy']
```

Reply:
fixed